cmd/kubeadm/app/preflight/checks.go

```diff
@@ -871,7 +871,7 @@ func (ncc NumCPUCheck) Check() (warnings, errorList []error) {
 	return warnings, errorList
 }
 
-// RunInitMasterChecks executes all individual, applicable to Master node checks.
+// RunInitMasterChecks executes all individual, applicable to master node checks.
 func RunInitMasterChecks(execer utilsexec.Interface, cfg *kubeadmapi.InitConfiguration, ignorePreflightErrors sets.String) error {
```

Comment:
looks like we missed this one...
could you please change it to?
`RunInitMasterChecks` -> `RunInitNodeChecks

Reply:
yes @neolit123,
I just modified variables, constants and function names under kubeadm/app/constants package. In addition, I also modified in the files which have reference to constants in kubeadm/app/constants.
I will change other package in next PR. Thanks

Reply:
ok, sounds good.
thanks for that!

/lgtm

Reply:
Ok @neolit123, I changed :)

Reply:
lol, I just updated PR :D

Reply:
:+1: let's wait to see if CI passes and will LGTM again.